migrations/mystery_items.js

```diff
@@ -27,6 +27,9 @@ if (_id) {
   db.users.update({_id:_id}, update);
 } else {
   // multiple (once @ start of event)
-  db.users.update({'purchased.plan.customerId':{$ne:null}}, update, {multi:true});
-}
-
+  db.users.update([{$and: [{'purchased.plan.customerId':{$ne:null}},
```

Comment:
No. The code sample I posted should be identical in content to your code, it just handles spacing and line breaks differently to make it more readable.